app/screens/auth/Leaving.tsx

```diff
@@ -88,12 +88,15 @@ const Leaving = ({ history }: AuthRouterParams) => {
           <ComplexLink>
             <Link onClick={navigateToSetupGuide} text="SETUP GUIDE" />
             <Row>
-              <Text>DON`T HAVE A DESKTOP?</Text>
+              <Text>DON`T HAVE A DESKTOP? &nbsp;</Text>
               <Link
-                onClick={() => history.push(AuthPath.Recover)}
+                onClick={() => history.push(AuthPath.Recover)} // Shouldn't it trigger set up instead of recover?
                 text="SETUP WALLET ONLY"
               />
-              <Tooltip width={100} text="SETUP WALLET ONLY" />
+              <Tooltip
+                width={140}
+                text="You will be able to send and receive SMH."
```

Comment:
And then probably this tooltip is excessive and we can just get rid of it.
Or at least write there about recovering options:
```suggestion
                text="You can recover your wallet from file or mnemonics"
```